src/modules/involvement.api.js

```diff
@@ -20,17 +20,17 @@ const setComment = async (movieID, comment) => {
   };
 
   const data = {
-    "item_id": movieID,
-    "username": getUser(),
-    "comment": comment
-  }
+    item_id: movieID,
+    username: getUser(),
+    comment,
+  };
   const appID = 'YUqI88f5a8VwBEjald5b';
   const res = await fetch(`https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/${appID}/comments`, {
     method: 'POST',
-    body: JSON.stringify(data)
+    body: JSON.stringify(data),
   });
```

Comment:
you need to add the content type
```
body: JSON.stringify(data),
headers: {
      'Content-type': 'application/json; charset=UTF-8',
    },
```

Reply:
Thanks for the tip :100:
I have added the headers, now we can add new comments